cups/ipp-file.c

```diff
@@ -703,6 +696,11 @@ parse_value(_ipp_file_t      *f,	/* I  - IPP data file */
 		    report_error(f, v, user_data, "No Language Data in line %d of \"%s\".", f->linenum, f->filename);
 		    return (0);
 		  }
+      if (!(value[0] == '(' && value[strlen(value)-1] == ')'))
```

Comment:
Don't you need to read another token here?

Reply:
Hi!
For input like "my string"(language), the var, value, at line 695, initially contains 'my string' , which then is fed into string.text. 
Next a token i.e. (language) is read into var value at line 696, which then is fed into string.language.
I don't think another token needs to be read here, is it ?